app/presenters/sufia/work_show_presenter.rb

```diff
@@ -2,6 +2,7 @@ module Sufia
   class WorkShowPresenter < ::CurationConcerns::WorkShowPresenter
     # delegate fields from Sufia::Works::Metadata to solr_document
     delegate :based_near, :depositor, :identifier, :resource_type, :tag, to: :solr_document
+    attr_reader :upload_set_id
```

Comment:
I don't know. I only added here because view `_script_templates.html.erb` (see below) references it.